### PR TITLE
Updated Nodes, Entities, Custom tools and Intro. md files and images

### PR DESCRIPTION
**Pull request by nikunj1705**
- Added Introduction for Athena and Mogambo
- Updated Nodes, Entities in Mogambo
- Updated Adding Custom tools (Athena)
- Uploaded ie replaced new images wherever relevant

### CODE REVIEW

File: docs/agent-chat/index.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
 ---
-title: Getting started
+title: Introduction
 ---
 
-Chat Solutions at Haptik can be powered entirely by Artificial Intelligence(AI) or by a combination of AI and Chat Agents.
+Chat Solutions at Haptik can be powered entirely by Artificial Intelligence (AI) or by a combination of AI and Chat Agents. We have built a tool for it, called "**Athena**". This tool is created for operationally managing user chats.
```
Comment (GanadiniAkshay) [inline docs/agent-chat/index.md]: Don't use internal names. Only Bot Builder, Analytics Tool or Agent Chat

File: docs/agent-chat/index.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
 
-Chat flows are built by the Haptik product team to best cater to the needs of the client and the solution being deployed. Chat flows are designed to guide the user through a specific problem and are the most important factor to ensure the best experience for all users.
+## Using Athena
```
Comment (GanadiniAkshay) [inline docs/agent-chat/index.md]: same here, don't use internal terms

File: docs/agent-chat/index.md [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
 
-In case a bot is unable to respond to a query which is outside the chat flow, the chat is handed over to a chat agent. Agents receive and send message over an web based tool called Athena. Athena has the same capabilities of the AI along with additional functionality for smooth operational deployment such as monitoring, review, assignment of chats by managers etc.
+To use this tool, users are to be created with a specific set of permissions. The users can then start monitoring and taking daily chats on their business. *Athena* has the same capabilities of the AI along with additional functionalities such as chat flow analytics, assignment of chats by managers etc.
```
Comment (GanadiniAkshay) [inline docs/agent-chat/index.md]: Athena has same capabilities as the AI?

File: docs/bot-builder/basic/introduction.md [hunk 4]
```diff
@@ -2,14 +2,26 @@
 title: Introduction
 ---
 
-Welcome to Mogambo, Haptik's bot builder tool! Once you've mastered this software, you'll be able to build chatbots with minimal coding. To get you started, we've put together some materials that will guide you through the process of building a bot on Mogambo. Depending on your learning preference, you can pick and choose from the following forms of documentation:
+Welcome to **Mogambo**, Haptik's bot builder tool! Once you've mastered *Mogambo*, you'll be able to build bots with minimal coding. To get you started, we've put together this documentation that will guide you through the process of building a bot on *Mogambo*.
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/introduction.md]: Don't use internal terms

File: docs/bot-builder/basic/introduction.md [hunk 5]
```diff
@@ -2,14 +2,26 @@
...
 
-- **Written Instructions**: We've detailed out the process of creating bots step-by-step, all the way from chatflow design to bot deployment. Keep reading for more.
-- **Video Tutorials**: Find videos outlining some of the more complicated processes in bot building HERE.
-- **Tool Tips**: For those of you who want to jump right in, we've built tooltips directly into Mogambo. By hovering over the tooltips, which are indicated by the small "i" in circles, you'll be able to view brief descriptions of how to use most of the major tools in Mogambo. Most tool tips also give links to the relevant corresponding sections in our written instructions, in case you need more information.
-- **Glossary**: A glossary of all the terms you might encounter over the course of building a bot. Find that HERE.
-- **Design Tips**: Creating the conversation flow your bots will use might seem straightforward, but there are a few things bot builders can do to increase how engaged users feel as they interact with bots. Read through the Haptik designers' pro tips HERE.
+    Tip: A bot is a program that communicates with you, commonly using a chat or voice interface.
+
+**How is Mogambo used?**
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/introduction.md]: Don't use internal terms

File: docs/bot-builder/basic/introduction.md [hunk 6]
```diff
@@ -2,14 +2,26 @@
...
+On *Mogambo*, you can create a bot i.e. formulate different conversation flows. Instead of creating bots on your own, *Mogambo* gives you a framework and components to build the bot without much effort. 
+
+**How does the Bot building process work?** 
+Creating the conversation flow in your bots might seem straightforward, but there are a few things bot builders do to engage users as they interact with bots - 
+
+1. You add the expected user intents (*user says*) and corresponding bot responses (*bot says*)
+2. And then, you configure ways a bot is supposed to interpret user's inputs. *Integrations* and *Settings* are components within *Mogambo* to configure your bot
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/introduction.md]: Internal term

File: docs/bot-builder/basic/making-first-bot.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ Fill the bot details according to the instructions given here:
...
+- **Test Business**: If you aren't yet familiar with the organizational system used to classify bots, we recommend you read the full overview [here](https://docs.haptik.ai/bot-builder/basic/bot-hierarchy) before continuing with your bot building. _As a refresher, a [business](https://docs.haptik.ai/bot-builder/basic/business) is a single bot or cluster of bots that can be deployed across a single partner or multiple partners._ This field is where you'll set the test business for your bot, which is a prerequisite to using the Test Chat feature in the bot building graph. When you test your bot in the graph, you'll be able to test not just the individual bot you're working on, but also the other bots that are live within a business that's been deployed on a particular partner. Because bots can be deployed across multiple businesses and certain business level chat settings can be customized for the different partners it's live on. Setting a test business becomes mandatory because this determines which business/settings the test chat feature will allow users to interact with.
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/making-first-bot.md]: _As should be As

Comment (GanadiniAkshay) [reply docs/bot-builder/basic/making-first-bot.md]: extra _ after partners.

File: docs/bot-builder/basic/making-first-bot.md [hunk 8]
```diff
@@ -22,7 +22,7 @@ Fill the bot details according to the instructions given here:
...
 
 - **Test Partner**: Just like businesses, partners are another component of the platform's organizational hierarchy. _In short, a partner is the bot building tool's largest unit of organization, and is a single business or cluster of businesses that can be deployed in multiple locations (i.e. a website, Facebook Messenger, and so forth)._ For the same reason as above, users must set a test partner to make it possible to use the Test Chat feature in the bot building graph.
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/making-first-bot.md]: _In should be In

File: docs/agent-chat/index.md (hunk 3 above)
Comment (nikunj1705) [reply docs/agent-chat/index.md]: Yes. The bot is used initially, where our ML models are in place. And later this chat moves to the agent.

File: docs/bot-builder/basic/introduction.md [hunk 9]
```diff
@@ -2,25 +2,25 @@
...
+2. And then, you configure ways a bot is supposed to interpret user's inputs. *Integrations* and *Settings* are components within *Bot Builder* to configure your bot
 3. Once you create the bot, you test it before deploying across any platform. 
 
 We've detailed out the process of creating bots step-by-step, all the way from chatflow design to bot deployment. Read up the various articles from the left-hand navigation to know more. 
 
-    Tool Tips: For those of you who want to jump right in, we've built tooltips directly into Mogambo. 
+    Tool Tips: For those of you who want to jump right in, we've built tooltips directly into Bot Builder. 
     By hovering over the tooltips, which are indicated by the small "i" or "?" in circles, you'll be able 
-    to view brief descriptions of how to use most of the major components in Mogambo. 
+    to view brief descriptions of how to use most of the major components in Bot maker.
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/introduction.md]: Bot Builder

**Pull request merged**